velox/functions/prestosql/tests/CastBaseTest.h

```diff
@@ -236,7 +236,7 @@ class CastBaseTest : public FunctionBaseTest {
     testCast(fromType, toType, inputVector, expectedVector);
   }
 
-  template <typename TFrom, typename TTo>
+  template <typename TFrom>
```

Comment:
Remove `typename TTo` as it is not in use. Do I need to do this refactor in a separate PR?

Reply:
This is a nice change. It would be great to extract it into a separate PR to make reviewing easier and faster. Thanks.

Reply:
Extract this change to https://github.com/facebookincubator/velox/pull/8464. Thanks.